test_utilities/pubspec.yaml

```diff
@@ -5,6 +5,9 @@ homepage: https://github.com/flutter/cocoon
 
 environment:
   sdk: '>=2.2.0'
+# Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
I think this got pasted on the wrong line :)

Reply:
Fixed!